app/views/reviews/new.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="row">
+  <%- if @review.errors.present? -%>
+    <div class="large-12 alert-box alert" data-alert>
+      <ul>
+        <%- @review.errors.full_messages.each do |error| -%>
+          <li><%= error %></li>
+        <%- end -%>
+      </ul>
+    </div>
+  <%- end -%>
+</div>
```

Comment:
Instead of needing to put this code at the top of every form, use flash from your controller to flash the error messages, and then you won't need this chunk!